examples/local/backups/take_backups.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Copyright 2022 The Vitess Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# this script brings up new tablets for the two new shards that we will
+# be creating in the customer keyspace and copies the schema
+
+source ./env.sh
+
+vtctlclient BackupShard -allow_primary=true customer/-80
```

Comment:
The clusters we start have 3 tablets each. Is there a reason we need to pass `-allow_primary`?

Reply:
I don't think it should be required. The command first tries to find a tablet which is of type REPLICA or RDONLY. If it doesn't find anything then it checks that flag and selects a PRIMARY tablet. If we have a running REPLICA tablet, then it should not matter wether we provide this flag or not.

Reply:
Right, thanks for the explanation on how the flag works internally! I will try removing it